integration-tests/README.md

```diff
@@ -15,24 +15,27 @@ TERRA_TOKEN=[bearer token] LYLE_SA_KEY=[full key json] yarn test
 #### Configuration
 The following environment variables are parsed by the tests:
 
-1. `TERRA_TOKEN` **(required)**: bearer token used to log into Terra.
-2. `TEST_URL`: URL for the ui.
+1. `TERRA_SA_KEY` **(required\*)**: service account key for creating access tokens for test user.
+1. `TERRA_TOKEN` **(required\*)**: bearer token used to log into Terra.
```

Comment:
I was going to have to change all of the numbers anyway, so I'm making them all `1.` and letting markdown number them correctly so that future diffs can be smaller.

Reply:
I'd kind of like them all to be labeled 0, in that case, since it's clearer when reading the unrendered version, as that's not a number we'd use.

Reply:
(Either that, or just make it an unordered list, since order doesn't matter)

Reply:
Oh, both good calls. I'm going to go with unordered list.